src/triage/experiments/validate.py

```diff
@@ -290,15 +279,25 @@ def _validate_query(self, query):
                 query: "{}"
                 Full error: {}'''.format(query, e)))
 
+    def _validate_include_missing_labels_in_train_as(self, missing_label_flag):
+        if missing_label_flag not in set([None, True, False]):
+            raise ValueError(dedent('''Section: label_config -
+            The value for 'include_missing_labels_in_train_as', {}, is invalid.
+            The key must be either absent, or a boolean value True or False
+            Triage only supports binary labels at this time.'''.format(missing_label_flag)))
```

Comment:
`dedent` isn't going to work with the message formatted this way; the indentation has to be consistent across lines.

    In [7]: if True:
       ...:     if True:
       ...:         raise ValueError(dedent('''Section: label_config -
       ...:         The value for 'include_missing_labels_in_train_as', {}, is invalid.
       ...:         The key must be either absent, or a boolean value True or False
       ...:         Triage only supports binary labels at this time.'''.format(missing_label
       ...: _flag)))
       ...: 
       ...: 
    ERROR:root:An unexpected error occurred while tokenizing input
    The following traceback may be corrupted or invalid
    The error message is: ('EOF in multi-line string', (1, 56))

    ---------------------------------------------------------------------------
    ValueError                                Traceback (most recent call last)
    <ipython-input-7-d00e60ed48c2> in <module>()
          4         The value for 'include_missing_labels_in_train_as', {}, is invalid.
          5         The key must be either absent, or a boolean value True or False
    ----> 6         Triage only supports binary labels at this time.'''.format(missing_label_flag)))
          7 

    ValueError: Section: label_config -
            The value for 'include_missing_labels_in_train_as', foo, is invalid.
            The key must be either absent, or a boolean value True or False
            Triage only supports binary labels at this time.

And, for something like this, why bother? _E.g.:_

    raise ValueError('Section: label_config\n\n'
                     f"The value for 'include_missing_labels_in_train_as', {missing_label_flag!r}, is invalid. "
                     'The key must be either absent, or a boolean value True or False.\n\n'
                     'Triage only supports binary labels at this time.')

…which comes out to:

    ValueError: Section: label_config

    The value for 'include_missing_labels_in_train_as', 'foo', is invalid. The key must be either absent, or a boolean value 
    True or False.

    Triage only supports binary labels at this time.